src/apps/loyalty/server/index.tsx

```diff
@@ -31,7 +32,9 @@ const {
 
 app.use(artsyPassport(Object.assign({
   CurrentUser: Backbone.Model,
+  ARTSY_URL: API_URL,
```

Comment:
artsy passport defaults to `ARTSY_URL`, and thus requires this overload, which was the root of the confusion